cli/src/lib/rpc-client.ts

```diff
@@ -0,0 +1,63 @@
+import type { Network } from "./constants.js";
+import {
+  type Abi,
+  createPublicClient,
+  decodeFunctionResult,
+  encodeFunctionData,
+  type Hex,
+  http,
+} from "viem";
+import type { TypeOf, ZodType } from "zod";
+
+const DEFAULT_URLS = {
+  mainnet: "https://ethereum-rpc.publicnode.com",
+  sepolia: "https://ethereum-sepolia-rpc.publicnode.com",
+};
+
+export class RpcClient {
```

Comment:
We used to have to functions to do rpc calls. I decided to create this class to be able to hold configuration regarding rpc calls.

Reply:
The old functions were inside "contract-read.ts" (that file was deleted) and they were called `contractRead` and `contractReadRaw`.